### PR TITLE
add com.xunlei.Thunder

### PR DESCRIPTION
**Pull request by qvshuo**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build com.xunlei.Thunder

File: com.xunlei.Thunder.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  <id>com.xunlei.Thunder</id>
+  <launchable type="desktop-id">com.xunlei.Thunder.desktop</launchable>
+  <name>Thunder</name>
+  <developer_name>xunlei</developer_name>
+  <summary>Xunlei download. 迅雷下载。</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://i.xunlei.com/xluser/legal/agreement.html</project_license>
+  <url type="homepage">https://www.xunlei.com/</url>
+  <description>
+    <p>Xunlei download, which supports downloading HTTP, BitTorrent, Magnet, etc.</p>
```
Comment (nedrichards) [inline com.xunlei.Thunder.appdata.xml:12]: Could you include a note that this is not supported by upstream, similar to this one for Slack. Other than that, lgtm

https://github.com/flathub/com.slack.Slack/blob/master/com.slack.Slack.appdata.xml#L18

Comment (qvshuo) [reply com.xunlei.Thunder.appdata.xml:12]: Thank you for the suggestion, I have added it through [this commit](https://github.com/qvshuo/flathub/commit/a441ac3179ef33a5fb6c5df55532982a301c9d60).

**Comment by qvshuo** [comment]
bot, build com.xunlei.Thunder

**Pull request closed without merging**